src/components/base/Input/Input.tsx

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+
+interface InputProps {
+  width: number;
+  height: number;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  name: string;
+  error: boolean;
+}
```

Comment:
```suggestion
interface InputProps {
  width: number;
  height: number;
  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
  value: string;
  name: string;
  error: boolean;
}
```
value는 controlled input에는 꼭 필요한 속성이니까 명시적으로 적어주면 좋을 것 같아요!